src/data/data.js

```diff
@@ -80,168 +93,170 @@ const pageData = {
       {
         name: 'headerTitleComp',
         title: 'Contexto',
-        header: 'El ACU'
+        header: 'El ACU',
       },
       {
         name: 'textBlockComp',
         backgroundColor: 'white',
         text:
-          'En todos los hogares se preparan alimentos y en la elaboración de muchos de ellos se emplea el aceite vegetal o animal. Luego de ser usado se conoce como ACU; Aceite de Cocina Usado. Este producto al ser utilizado varias veces, llega un momento en que se quema y es desechado muchas veces de manera incorrecta. Lo que contamina el medio ambiente, tanto suelos como aguas.'
+          'En todos los hogares se preparan alimentos y en la elaboración de muchos de ellos se emplea el aceite vegetal o animal. Luego de ser usado se conoce como ACU; Aceite de Cocina Usado. Este producto al ser utilizado varias veces, llega un momento en que se quema y es desechado muchas veces de manera incorrecta. Lo que contamina el medio ambiente, tanto suelos como aguas.',
       },
       {
         name: 'percentTimelineHeader',
-        title: 'Según nuestro estudio:'
+        title: 'Según nuestro estudio:',
       },
       {
         name: 'percentTimelineItem',
         percent: '30%',
         title: 'Lo tiran por el desagüe',
-        text: 'Contaminando ríos y depósitos de agua, creando focos de hongos y bacterías.',
-        image: ''
+        text:
+          'Contaminando ríos y depósitos de agua, creando focos de hongos y bacterías.',
+        image: ',
       },
       {
         name: 'percentTimelineItem',
         percent: '20%',
         title: 'Lo tiran a la basura',
         text: 'Filtrándose en suelos, poniendo en riesgo la vida al rededor.',
-        image: ''
+        image: ',
```

Comment:
Same as before. Don't remove the closing quote.

Reply:
resolved